build_tools/python/e2e_test_framework/definitions/iree_definitions.py

```diff
@@ -101,14 +101,16 @@ class ImportedModel(object):
   """Describes an imported MLIR model."""
   model: common_definitions.Model
   dialect_type: MLIRDialectType
+  import_flags: str
 
   @staticmethod
   def from_model(model: common_definitions.Model):
     # Currently we assume the model source type and its imported dialect is an
     # 1-1 mapping.
     return ImportedModel(
         model=model,
-        dialect_type=MODEL_SOURCE_TO_DIALECT_TYPE_MAP[model.source_type])
+        dialect_type=MODEL_SOURCE_TO_DIALECT_TYPE_MAP[model.source_type],
+        import_flags=f"--tf-savedmodel-exported-names={model.entry_function}")
```

Comment:
This will populate the flags for all models, including TFLite. You can check the `model.source_type` to only set flags on the TF source model.